bears/python/PyUnusedCodeBear.py

```diff
@@ -15,16 +15,16 @@ class PyUnusedCodeBear(LocalBear):
     CAN_DETECT = {'Unused Code'}
 
     def run(self, filename, file,
-            remove_all_unused_imports: bool=False,
+            remove_all_unused_imports: bool=True,
             remove_unused_variables: bool=True):
         """
         Detects unused code. By default this functionality is limited to:
 
         - Unneeded pass statements.
-        - Unneeded builtin imports.
+        - All unneeded imports.
```

Comment:
According to @meetmangukiya's https://github.com/coala/coala-bears/pull/1829#discussion_r122533528 , and in contrast to my earlier https://github.com/coala/coala-bears/pull/1844#discussion_r122526269 ;) , I think now that it's better to completely change *unneeded* to *unused* in the docstring :)

cc @jayvdb